CHANGELOG.md

```diff
@@ -1 +1 @@
-The changelog has moved to https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/changelog.html
+The changelog has moved to https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/16.x/changelog.html
```

Comment:
This URL doesn't contain any information.

Reply:
Hmm, all the info seems to have moved to the sections outlined on the right, I'll see if there's something we can do to get these listed back into a single page.